lib/connection.js

```diff
@@ -1,12 +1,126 @@
 'use strict';
 
 const common = require('metarhia-common');
-const EventEmitter = require('events');
+const { EventEmitter } = require('events');
```

Comment:
Just the same, `requite('events')` returns `EventEmitter` but it have circular reference